packages/coding-agent/test/exit-command.test.ts

```diff
@@ -0,0 +1,94 @@
+import { describe, expect, test } from "vitest";
+
+describe("Exit command", () => {
```

Comment:
Tell me you didn't read the vibed code without telling me you didn't read the vibed code :)

What is this supposed to test? That V8's startsWith works as intended?